docs/userguide/configuring.rst

```diff
@@ -75,9 +77,11 @@ Consider a configuration function if the application will shut down and re-launc
     def make_config() -> Config:
         return Config(...)
 
-    parsl.load(make_config())
+    with parsl.load(make_config()):
+        pass
```

Comment:
it would be better to write something like ' # Your workflow here' instead of 'pass'